apps/mobile/package.json

```diff
@@ -90,6 +90,7 @@
     "react-native-device-info": "^14.0.1",
     "react-native-draggable-flatlist": "^4.0.1",
     "react-native-gesture-handler": "~2.20.2",
+    "react-native-get-random-values": "^1.11.0",
```

Comment:
before we add this package I would actually want to know why is quick-crypto not doing this. Maybe we are calling the polyfills from it too late?